cobbler/modules/managers/isc.py

```diff
@@ -24,59 +24,51 @@
 import time
 import copy
 
-from cobbler import templar
-from cobbler import utils
+import cobbler.utils as utils
+from cobbler.manager import ManagerModule
 
 from cobbler.cexceptions import CX
 
 
-def register() -> str:
+def register():
     """
     The mandatory Cobbler module registration hook.
     """
     return "manage"
 
 
-class IscManager:
+class _IscManager(ManagerModule):
 
-    def what(self) -> str:
+    @staticmethod
+    def what():
         """
         Static method to identify the manager.
 
         :return: Always "isc".
         """
         return "isc"
 
+    # ToDo: Get rid of logger
     def __init__(self, collection_mgr, logger):
-        """
-        Constructor
+        super().__init__(collection_mgr, logger)
 
-        :param collection_mgr: The collection manager to resolve all information with.
-        :param logger: The logger to audit all actions with.
-        """
-        self.logger = logger
-        self.collection_mgr = collection_mgr
-        self.api = collection_mgr.api
-        self.distros = collection_mgr.distros()
-        self.profiles = collection_mgr.profiles()
-        self.systems = collection_mgr.systems()
-        self.settings = collection_mgr.settings()
-        self.repos = collection_mgr.repos()
-        self.templar = templar.Templar(collection_mgr)
-        self.settings_file = utils.dhcpconf_location()
-
-    def write_dhcp_file(self):
+        self.settings_file = utils.dhcpconf_location(self.api)
+        self.settings_file_v6 = '/etc/dhcpd6.conf'
+
+    def write_v4_config(self):
         """
-        DHCP files are written when ``manage_dhcp`` is set in our settings.
+        DHCP files are written when ``manage_dhcp`` is set in ``/etc/cobbler/settings``.
         """
 
         template_file = "/etc/cobbler/dhcp.template"
         blender_cache = {}
 
         try:
             f2 = open(template_file, "r")
-        except:
+
+        except Exception:
```

Comment:
That is too lazy. Catch the exception you want to catch and handle them individually. Otherwise let it crash. Just silencing and re-rasing is not good style.